lib/project_types/theme/commands/language_server.rb

```diff
@@ -9,7 +9,7 @@ def call(*)
       end
 
       def self.help
-        ShopifyCli::Context.message("theme.language_server.help", ShopifyCli::TOOL_NAME)
+        ShopifyCli::Context.message("theme.language_server.help", ShopifyCli::TOOL_NAME, ShopifyCli::TOOL_NAME)
```

Comment:
An alternative to having to use the same parameter twice is to use `%1$s` in the `theme.language_server.help` message, i.e., in `theme/messages/messages.rb`:
```
        language_server: {
          help: <<~HELP,
            {{command:%1$s theme language-server}}: Start a Language Server Protocol server.

            Usage: {{command:%1$s theme language-server}}
          HELP
        },
```
and then the line above can remain
```
ShopifyCli::Context.message("theme.language_server.help", ShopifyCli::TOOL_NAME)
```